packages/decorators/src/decorators/http/http-function.ts

```diff
@@ -1,10 +1,47 @@
 import { handleRequestBodyParameter } from './request-body';
 import { handleQueryParameters } from './query-parameter';
 import { handlePathParameter } from './path-parameter';
-import { isContext, isFunction, isHttpRequest } from './type-guards';
+import { isContext, isFunction, isHttpRequest, isHttpResponse } from './type-guards';
 import { handleContextParameter } from '../context';
 import { handleRequestParameter } from './http-request';
 import { handleError } from './http-status';
+import { HttpResponse } from '@azure/functions';
+
+type ResultMapper<T> = (result: T) => HttpResponse;
+
+type FullHttpFunctionOptions = {
+    ResultMapper: ResultMapper<any>;
+};
+
+type HttpFunctionOptions = Partial<FullHttpFunctionOptions>;
+
+const defaultResultMapper: ResultMapper<unknown> = (result: unknown): HttpResponse => {
+    if (!result) {
```

Comment:
```suggestion
    if (result === undefined || result === null) {
```

Otherwise, primitive `0` and `false` would end up here.